xline/src/id_gen.rs

```diff
@@ -38,6 +38,11 @@ impl IdGenerator {
         let suffix = self.suffix.fetch_add(1, Ordering::Release);
         self.prefix | suffix
     }
+
+    /// Generate next id and cast to i64
+    pub(crate) fn next_as_i64(&self) -> i64 {
```

Comment:
Why don't you inline the `next` method here and rename the `next_as_i64` as `next`? Doing so can increase coverage.

Reply:
Fixed